docs/02-concepts/01-data-processing/04-batch-jobs.md

```diff
@@ -114,8 +115,9 @@ To fit into as many environments as possible, a mechanism is provided to
 extract the consent from the source data:
 
 1. Specify the field that contains the consent
-2. Specify a mapping to translate your terminology to consent
-   levels (which are integer arrays). A mapping in pseudo-code could be:
+2. Specify a mapping to translate your terminology to purposes from your [purpose maps][purpose-maps].
+   Provide the integer values ("levels") of the applicable purposes.
```

Comment:
The fact that `purposes` is plural implies that there is an option to bind multiple levels to a single purpose, which is not the case.